app/controllers/state_file/questions/pending_federal_return_controller.rb

```diff
@@ -1,6 +1,6 @@
 module StateFile
   module Questions
-    class PendingFederalReturnController < QuestionsController
+    class PendingFederalReturnController < LandingPageController
```

Comment:
May need a bigger refactor to make this PR and a few other pages `namespaced` entirely to LandingPages instead of to Questions.